spec/vcr/api/supplementals_controller/show/supplemental_application.yml

```diff
@@ -528,3164 +528,6 @@ http_interactions:
     body:
       encoding: ASCII-8BIT
       string: '{"totalSize":0,"done":true,"records":[]}'
-    http_version: null
-  recorded_at: Wed, 09 Dec 2020 19:45:56 GMT
```

Comment:
I'm confused by why so many lines were deleted in this file, doesn't seem like there were so many changes to this request?

Reply:
looking at it again, I think that it changed when we moved to making separate requests for different supp app pieces instead of including it all in the same request. And we just didn't refresh the cassette then + it happened to not break the tests 🤪 . Either in https://github.com/SFDigitalServices/sf-dahlia-lap/pull/469 or https://github.com/SFDigitalServices/sf-dahlia-lap/pull/471